docs/conf/prague/2021/news/tickets-cfp-reminder.rst

```diff
@@ -0,0 +1,66 @@
+:template: {{year}}/generic.html
+
+.. post:: May 25, 2021
+   :tags: prague-2021, cfp, tickets
+
+Announcing Ticket Sales, Grant Program, and Call for Proposals Reminder
+=======================================================================
+
+Greetings, documentarians! Today we're happy to announce that tickets are `on sale now <https://www.writethedocs.org/conf/prague/{{year}}/tickets/>`_, and to remind you that our `Call for Proposals <https://www.writethedocs.org/conf/prague/{{year}}/cfp/>`_ is still open.
+
+Following the success of the 2020 virtual conference, we look forward to more opportunities to bring the community together virtually and to share the wonderful energy with each other wherever you are in the world.
+
+If there is something you’d really like to see a talk about this year, submit a proposal for it or refer someone else who would be good!
+
+Get your ticket for the conference
+----------------------------------
+
+**The Prague conference will be held online on {{ date.short }}.** The `full event website <https://www.writethedocs.org/conf/prague/{{year}}/>`_ is live, with some preliminary details, and we'll be filling in more as it gets closer.
+
+If you're excited too, go ahead and `register for your ticket now <https://www.writethedocs.org/conf/prague/{{year}}/tickets/>`_! Even though our capacity is more flexible due to the virtual event platform, we might still sell out to make sure we're providing a manageable event size for our attendees, as always.
+
+Similar to last year's virtual conference, we're going to keep as much as possible of the spirit of the event alive:
+
+* Excellent single track of pre-recorded presentations with live moderated Q&A after each talk
+* Writing Day sessions
+* Unconference sessions
+* Job Fair booths
+* Sponsor booths
```

Comment:
We should probably not commit to this, since we've never really had sponsor booths in Prague. Also, do we want to promise the job fair? Last year it was a scramble to try and fill it :/